### PR TITLE
Localdev doc

### PR DESCRIPTION
**Pull request by omendram**

### CODE REVIEW

**Comment by RubenGeo** [comment]
Thanks for the pull request Omendra it looks well structured and clear to me. The only things that are still missing are:

- The setup for MacOS
- Removing overlapping parts from other Readmes.

**Comment by omendram** [comment]
> Thanks for the pull request Omendra it looks well structured and clear to me. The only things that are still missing are:
> 
> * The setup for MacOS
> * Removing overlapping/redundant parts from other Readmes
> 
> Could you still add this?

Yes, I will add the parts and also remove the duplicate content in nested folders.

File: README.md [hunk 1]
```diff
@@ -27,12 +27,114 @@ The documentation of the 121 platform can be found on the Wiki of this repositor
...
+    cp 121-service/example.ormconfig.json 121-service/ormconfig.json
+    cp PA-accounts-service/src/example.secrets.ts PA-accounts-service/src/secrets.ts
+    cp PA-accounts-service/example.ormconfig.json PA-accounts-service/ormconfig.json
+
+Environment variables are explained in the comments of the .env.example
+
+
+You can start in the development mode by following:
 - Run: `npm run install:all` from *this* folder
 - Run: `npm run start:all` from *this* folder
```
Comment (diderikvw) [inline README.md:129]: Not clear what "this" refers to. I assume it means: run the commands from the services folder.

Comment (omendram) [reply README.md:129]: Adding the related clarification.

**Comment by diderikvw** [comment]
Structure of README should be improved: apply proper heading levels per section.

**Comment by omendram** [comment]
@RubenGeo I have made the changes according to the comments.

File: README.md [hunk 2]
```diff
@@ -27,11 +27,113 @@ The documentation of the 121 platform can be found on the Wiki of this repositor
...
+
+On linux distributions we need to install docker enginer and docker compose respectively. On other platforms docker-compose enginer is available through Docker Desktop. Read more at: https://docs.docker.com/engine/install/
+
+Docker compose relies on docker engine, so in order to get started we must install docker-engine first. (Read more at: https://docs.docker.com/engine/install/ubuntu/). Theere are various ways of installing docker engine (repository, packages or automated scripts). The official recommended way doing that is through setting up a repository.
+
+#### First off, uninstall any old version of docker-engine.
+
+    sudo apt-get remove docker docker-engine docker.io containerd runc
+
+#### Choose an installation method from the provided list at https://docs.docker.com/engine/install/ubuntu/#installation-methods and install docker-engine
```
Comment (diderikvw) [inline README.md:63]: It says to install docker-engine, but when opening the link and scrolling to "Install Docker Engine" that does not work. Installing docker is probably needed as well.

**Pull request merged**